samples/bluetooth/fast_pair/locator_tag/prj_release.conf

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
btw. Can we enable `CONFIG_ASSERT` in debug configuration to ensure correctness?

Reply:
Asserts are enabled in the debug configuration:

```
#define CONFIG_BT_ASSERT 1
#define CONFIG_BT_ASSERT_VERBOSE 1
#define CONFIG_ASSERT 1
#define CONFIG_ASSERT_LEVEL 2
#define CONFIG_ASSERT_VERBOSE 1
#define CONFIG_ASSERT_NO_COND_INFO 1
```

Reply:
I see now. It's implied by `CONFIG_NCS_SAMPLES_DEFAULTS`